models/issue.go

```
@@ -78,6 +93,14 @@ func (issue *Issue) loadRepo(e Engine) (err error) {
 	return nil
 }
 
+// IsTimetrackerEnabled returns true if the repo enables timetracking
+func (issue *Issue) IsTimetrackerEnabled() bool {
+	if issue.loadRepo(x) != nil {
+		return false
```

If error is ignored than at least it should be logged